templates/metric-alert.json

```diff
@@ -0,0 +1,189 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "subscription": {
+            "type": "string",
+            "defaultValue": "",
+            "minLength": 1,
+            "metadata": {
+                "description": "subscriptionID"
+            }
+        },
+        "actionGroupName": {
+            "type": "string",
+            "defaultValue": "s121d01-BAT-devops",
```

Comment:
remove the s121 specific `defaultValue`s as this template could be reused by other teams as well.